### ISSUE TITLE
Separate out SEM-SEG and OBJ-DET

### ISSUE THREAD

**Issue by Sabrina-Knappe** [Shared-Reality-Lab/IMAGE-browser#176]
On Unicorn, make it so that sem-seg and obj-det renderings are returned as separate entities. This is to be used during CSUN to get clearer feedback on how users are interpreting semantic segmentation.

Assigning to Juliette for clarifications on what the smaller work items are.

**Comment by JRegimbal** [comment]
We can either
1) Update the current combined handler to return additional single-preprocessor renderings,
2) Put the old handlers for each one back into production and update their code in SuperCollider to incorporate changes in the combined side, or
3) Update the old handlers to use the new more flexible responder for the combined handler.

3 would probably be the best option, but 1 is the quickest (even though it means that the renderings returned would not have the same content as in the combined handler).

**Comment by Sabrina-Knappe** [comment]
Tagging @Cybernide and @jeffbl since they will be the ones dealing with this at CSUN

**Comment by jeffbl** [comment]
I don't think we have REB for this yet, so I'm loathe to commit @JRegimbal 's limited time to it, since it would be 100% wasted if we end up not running this during CSUN. @Cybernide @Sabrina-Knappe do we have a solid plan for running this? (i.e., if this is implemented, and the REB comes through in time, is this key vs. things like implementing the user feedback system?) I'll be on gather this afternoon to discuss if necessary, esp. since I don't have a good handle on how much time this will take, and I don't understand why option number one wouldn't have the same individual rendering sections as the combined rendering.

**Comment by Sabrina-Knappe** [comment]
This is Cyan’s purview. I won’t be at the booth so I haven’t been working on the planning for feedback gathering.

From: Jeff Blum ***@***.***>
Date: Thursday, March 3, 2022 at 10:58 AM
To: Shared-Reality-Lab/IMAGE-browser ***@***.***>
Cc: Sabrina Knappe ***@***.***>, Mention ***@***.***>
Subject: Re: [Shared-Reality-Lab/IMAGE-browser] Separate out SEM-SEG and OBJ-DET (Issue #176)

I don't think we have REB for this yet, so I'm loathe to commit @JRegimbal<https://github.com/JRegimbal> 's limited time to it, since it would be 100% wasted if we end up not running this during CSUN. @Cybernide<https://github.com/Cybernide> @Sabrina-Knappe<https://github.com/Sabrina-Knappe> do we have a solid plan for running this? (i.e., if this is implemented, and the REB comes through in time, is this key vs. things like implementing the user feedback system?) I'll be on gather this afternoon to discuss if necessary, esp. since I don't have a good handle on how much time this will take, and I don't understand why option number one wouldn't have the same individual rendering sections as the combined rendering.

**Comment by Cybernide** [comment]
After meditating on this a bit, I think we can mitigate the underlying issue of not understanding semantic segmentation with better and more accessible (and I even mean conventionally accessible, as in understandable) end user documentation. And I think it's worth a try doing that instead of making a whole work item browser-side for it.

I had always meant to completely revamp the website and this is just giving me more reason to do it now. (Not to dunk on anyone's work on the website, it's just that things evolve.)
